src/parsers/pm_io/psse.jl

```diff
@@ -21,29 +21,33 @@ function _init_bus!(bus::Dict{String, Any}, id::Int)
     return
 end
 
+function _get_bus_value(bus_i::Int, field::String, pm_bus_data::Array)
+    for bus in pm_bus_data
+        if bus["index"] == bus_i
+            return bus[field]
+        end
+    end
+    @info("Could not find bus $bus_i, returning 0 for field $field")
+    return 0
+end
+
+function _get_bus_value(bus_i::Int, field::String, pm_bus_data::Dict{String, Any})
```

Comment:
The name `pm_bus_data` seems misleading. Should it be `pm_data` for consistency?

Reply:
I just named it like that since it is only the bus dict not the whole data

Reply:
It is the full pm_data, right? You have `pm_bus_data["bus"]`.